tools/grub/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM linuxkit/alpine:3683c9a66cd4da40bd7d6c7da599b2dcd738b559 AS grub-build
+FROM linuxkit/alpine:518c2ed0f398c5508969ac5e033607201fb419ed AS grub-build
```

Comment:
this should be `linuxkit/alpine:86cd4f51b49fb9a078b50201d892a3c7973d48ec`. The Alpine based was updated earlier this week

Reply:
I took the last tag from when I did this, probably Monday, as I flew on Tuesday. It is a bit slow to update it (earliest `FROM` line, causes an entire rebuild), will try to get it done.

Reply:
Updated, as well as the tags in the various `Dockerfile`, and `output.go`.